package.json

```diff
@@ -9,7 +9,6 @@
   "author": "Jordan Scales <scalesjordan@gmail.com>",
   "devDependencies": {
     "accessibility-developer-tools": "2.9.0-rc.0",
-    "autoprefixer-loader": "^2.0.0",
```

Comment:
Do we need to configure autoprefixing within postcss now? It seems some properties like `align-items` are not being prefixed. Can you confirm? 

![image](https://cloud.githubusercontent.com/assets/287268/20975439/8a1e5d88-bc6d-11e6-8d37-98feb5e5d560.png)

For reference, here's how `.tota11y-plugin-switch` is defined currently on master:

![image](https://cloud.githubusercontent.com/assets/287268/20975548/d313ad4a-bc6d-11e6-987c-4995f1f4593f.png)